### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by MartinJibber**

### CODE REVIEW

File: iOS/Flows/Circles/CircleCoordinator.swift [hunk 1]
```diff
@@ -68,6 +68,7 @@ class CircleCoordinator: PresentableCoordinator<Void> {
         let alertController = UIAlertController(title: "Update Name", message: "", preferredStyle: .alert)
         alertController.addTextField { (textField : UITextField!) -> Void in
             textField.placeholder = "Name"
+            textField.autocapitalizationType = .words
```
Comment (MartinJibber) [inline iOS/Flows/Circles/CircleCoordinator.swift:71]: This was annoying me lol

File: iOS/Flows/Conversation/Conversation Detail/ConversationDetailCoordinator.swift [hunk 2]
```diff
@@ -76,6 +76,7 @@ class ConversationDetailCoordinator: PresentableCoordinator<DetailCoordinatorRes
         let alertController = UIAlertController(title: "Update Name", message: "", preferredStyle: .alert)
         alertController.addTextField { (textField : UITextField!) -> Void in
             textField.placeholder = "Name"
+            textField.autocapitalizationType = .words
```
Comment (MartinJibber) [inline iOS/Flows/Conversation/Conversation Detail/ConversationDetailCoordinator.swift:79]: Same here

File: iOS/Flows/Conversation/Conversation Header/ConversationHeaderViewController.swift [hunk 3]
```diff
@@ -22,9 +22,6 @@ class ConversationHeaderViewController: ViewController, ActiveConversationable {
     let roomsButton = RoomNavigationButton()
     
     private var state: ConversationUIState = .read
-    
-    var didTapAddPeople: CompletionOptional = nil
-    var didTapUpdateTopic: CompletionOptional = nil
```
Comment (MartinJibber) [inline iOS/Flows/Conversation/Conversation Header/ConversationHeaderViewController.swift:27]: Not used any more

File: iOS/Flows/Conversation/Input/SwipeableInputAccessoryView.xib [hunk 4]
```diff
@@ -31,7 +31,7 @@
                                     <color key="backgroundColor" white="0.0" alpha="0.0" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
                                     <constraints>
                                         <constraint firstAttribute="height" relation="greaterThanOrEqual" constant="20" id="DYI-6o-w2X"/>
-                                        <constraint firstAttribute="height" relation="lessThanOrEqual" constant="82" id="uIT-mU-AJz"/>
+                                        <constraint firstAttribute="height" relation="lessThanOrEqual" constant="102" id="uIT-mU-AJz"/>
```
Comment (MartinJibber) [inline iOS/Flows/Conversation/Input/SwipeableInputAccessoryView.xib:34]: Fixes the textfield not growing big enough when you're at 3-4 lines.

File: Shared/UI/Views/MessageContentView/MessageContentView.swift [hunk 5]
```diff
@@ -18,6 +18,7 @@ class MessageContentView: BaseView {
 #if IOS
     var handleTappedMessage: ((ConversationId, MessageId) -> Void)?
     var handleEditMessage: ((ConversationId, MessageId) -> Void)?
+    var handleTappedAttachment: ((MediaItem) -> Void)?
```
Comment (MartinJibber) [inline Shared/UI/Views/MessageContentView/MessageContentView.swift:21]: Adding a handler var now, but not implementing it yet.

File: iOS/Flows/Conversation/Conversation Detail/ConversationDetailCoordinator.swift (hunk 2 above)
Comment (Benjamindodgson) [reply iOS/Flows/Conversation/Conversation Detail/ConversationDetailCoordinator.swift:79]: Nice

**Review by Benjamindodgson — APPROVED** [review]

**Pull request merged**